README.md

```diff
@@ -335,6 +335,8 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Bineet Naidu](https://github.com/bineetNaidu)
   - [bitM1ke](https://github.com/cryptome)
   - [Boby](https://github.com/bobychaudhary)
+  - [Bony](https://github.com/bonysureliya)
+
```

Comment:
There's an extra line here.
```suggestion
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable